src/Traits/EntryNote.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OWC\Zaaksysteem\Traits;
+
+use GFAPI;
+
+trait EntryNote
+{
+    public function entryAddNote($entryId, string $message, string $type = 'error'): void
+    {
+        GFAPI::add_note($entryId, 0, \OWC_GZ_SHORT_NAME, $message, 'user', $type);
```

Comment:
Ik zie in andere bestanden rechtstreekse calls naar `GFAPI`. Waarom zit deze method dan in een eigen trait?

Reply:
Omdat je nu vanuit andere classes deze methode kunt aanroepen met 3 parameters (waarvan de message al lang is). En inderdaad we zouden zonder de trait kunnen maar dit vind ik netter, vooral om te lange regels te voorkomen.